### PR TITLE
Add a Podspec for v0.0.1

### PR DESCRIPTION
**Pull request by mthole**

### CODE REVIEW

**Comment by mikelikespie** [comment]
LGTM

File: ThumborURL.podspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Pod::Spec.new do |s|
+  s.name         = "ThumborURL"
+  s.version      = "0.0.1"
```
Comment (kyleve) [inline ThumborURL.podspec]: Any reason this isn't 1.0?

**Comment by kyleve** [comment]
LGTM (Wow, Github doesn't have an approve button!)

File: ThumborURL.podspec (hunk 1 above)
Comment (mthole) [inline ThumborURL.podspec]: Pod specs start with the by default and I didn't change it. If we were to bump to 1.0, I'd wait to do so until a few housekeeping things (e.g. 64 bit warnings) are taken care of

**Pull request merged**